build_projects/dotnet-cli-build/DebTargets.cs

```diff
@@ -13,14 +13,14 @@ namespace Microsoft.DotNet.Cli.Build
     public class DebTargets
     {
         [Target(nameof(GenerateSdkDeb))]
-        [BuildPlatforms(BuildPlatform.Ubuntu)]
+        [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
This is currently all possible, but we need to do some work here to support publishing to a distro specific logical repo, which was set up by our friends in Azure Linux. Additionally we would need to set up the proper `debian_config.json` definiitions for 16.04 which should be straightforward.